### PR TITLE
:wrench: Fix typos in source code

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,./frontend/yarn.lock -L childs,clen,fpr,inflight,ody,ot,ro,te,trys,ue`

### CODE REVIEW

**Comment by luzpaz** [comment]
sorry, will rebase once I clone upstream correctly

**Comment by luzpaz** [comment]
Ok, should be ready for review

File: backend/src/app/util/time.clj [hunk 1]
```diff
@@ -41,7 +41,7 @@
       :hours   ChronoUnit/HOURS
       :days    ChronoUnit/DAYS
       :weeks   ChronoUnit/WEEKS
-      :monts   ChronoUnit/MONTHS)))
+      :months   ChronoUnit/MONTHS)))
```
Comment (luzpaz) [inline backend/src/app/util/time.clj:44]: please review closely

Comment (niwinz) [reply backend/src/app/util/time.clj:44]: yep, agree, but I'm pretty sure this fixes other silent bug.
I guess we don't use `:months` on the code.
:heart:

**Comment by niwinz** [comment]
Merged manually, thanks.

**Pull request closed without merging**